tests/filecheck/parser-printer/affine_map.mlir

```diff
@@ -27,4 +27,25 @@ builtin.module {
 
   // CHECK: "f9"() {"map" = #affine_map<(d0, d1, d2)[s0, s1, s2] -> ((d0 + s0), (d1 + s1), (d2 + s2))>} : () -> ()
   "f9"() {map = affine_map<(d0, d1, d2)[s0, s1, s2] -> (d0 + s0, d1 + s1, d2 + s2)>} : () -> ()
+
+  // CHECK: "f10"() {"map" = #affine_map<(d0, d1) -> (((d1 + (d0 * -1)) + ((((((d0 + (d1 * -1)) + 1) * 2) + d1) + -1) + 0)), (((((d1 + d1) + 1) + d1) + d1) + 1))>} : () -> ()
+  "f10"() { map = affine_map<(i, j) -> ((j - i) + 2*(i - j + 1) + j - 1 + 0, j + j + 1 + j + j + 1)> } : () -> ()
+
+  // CHECK: "f11"() {"map" = #affine_map<(d0, d1) -> ((((d0 + 4) * -1) + 6), d1)>} : () -> ()
+  "f11"() { map = affine_map<(i, j) -> (3+3-2*2+i, j)>} : () -> ()
```

Comment:
This looks wrong to me. Shouldn't the initial thing translate to 2+i? It seems to translate to 2-i, right?

Reply:
the whole thing is weird the expression in the check seems more complex than the one in the source...

Reply:
Oops, this was a parser bug. Fixed now.